apps/andi/lib/andi/input_schemas/display_names.ex

```diff
@@ -23,6 +23,7 @@ defmodule Andi.InputSchemas.DisplayNames do
     schema: "Schema",
     sourceFormat: "Source Format",
     sourceType: "Source Type",
+    sourceUrl: "Source Url",
```

Comment:
Not sure this is visible anywhere yet, but it probably should be `Source URL`.

Reply:
Changed